quickwit-ui/app/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "app",
+  "version": "0.1.0",
+  "license" : "AGPL-3.0",
+  "private": true,
+  "dependencies": {
+    "@monaco-editor/react": "^4.3.1",
+    "@mui/icons-material": "^5.3.0",
+    "@mui/lab": "^5.0.0-alpha.65",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^12.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/jest": "^27.0.1",
+    "@types/node": "^16.7.13",
+    "@types/react": "^17.0.20",
+    "@types/react-dom": "^17.0.9",
+    "dayjs": "^1.10.7",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-number-format": "^4.9.1",
+    "react-scripts": "5.0.0",
+    "styled-components": "^5.3.3",
+    "styled-icons": "^10.45.0",
+    "typescript": "^4.4.2",
+    "web-vitals": "^2.1.0"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
```

Comment:
config, but no eslint in the dependencies?

Reply:
no ideas, I'm so lazy with the js stack...

Reply:
we should add an eslinter